integration/fuse/bin/alluxio-fuse

```diff
@@ -118,7 +119,19 @@ fuse_stat() {
   fi
 }
 
-USAGE_MSG="Usage:\n\talluxio-fuse [mount|umount|stat]"
+USAGE_MSG="Usage:\n\talluxio-fuse [mount|umount|stat]
+
+mount \tmount an Alluxio path to local file system
+umount \tdismount an Alluxio path from local file system
+stat \tshow status of Alluxio mount points
+"
+
+MOUNT_USAGE="Usage: \talluxio-fuse mount [-n] [-o <mount option>] <mount_point> [alluxio_path]
```

Comment:
I think adding the tab in `Usage: \t...` is unnecessary. The usage statement isn't in a table format. It will also probably be better for line length to just use a single space. Either that or use `\n\t`. It seems to be the format of most GNU coreutils (`cp`, `rm`, `mv`, `mount`, etc..)

Reply:
use `\n\t` consistently in this file